openquake/qa_tests_data/classical/case_90/sites.csv

```diff
@@ -0,0 +1,5 @@
+lon,lat,vs30,z1pt0,z2pt5
+-122,37.95,681.59,72.1,0.69
+-121.65,37.15,893.99,-1,-1
+-122.25,37.8,301.29,457.77,1.75
+-122.05,37.8,653.4,-1,-1
```

Comment:
You do not need to write an entire new case_90 just to test the ValueError, the check can be included in case_89, I can show you how.